python/lsst/ts/analysis/notebooks/analysisTools.py

```diff
@@ -0,0 +1,399 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from astropy.visualization import ZScaleInterval
+from lsst.daf import butler as dafButler
+
+from matplotlib import rcParams
+
+rcParams["ytick.labelsize"] = 15
+rcParams["xtick.labelsize"] = 15
+rcParams["axes.labelsize"] = 20
+rcParams["axes.linewidth"] = 2
+rcParams["font.size"] = 15
+rcParams["axes.titlesize"] = 18
+
+# common functions for AOS analysis
+
+
+def plotZernikeImageFromPickle(
+    expIntra,
+    yearMonthDay,
+    zkResultsFile=None,
+    collection="u/scichris/Latiss/postISRex",
```

Comment:
This looks particular to you own environment ("scichris").. Maybe leave it with no defaults?